curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/665076b10b3d9236441b6437.md

```diff
@@ -0,0 +1,54 @@
+---
+id: 665076b10b3d9236441b6437
+title: Step 22
+challengeType: 1
+dashedName: step-22
+---
+
+# --description--
+
+Now you are ready to move onto the next set of array lessons.
+
+Remove all of your code from the previous step.
+
+# --hints--
+
+You should not have a `cities` array in your code.
+
+```js
+assert.notMatch(code, /let\s+cities\s*=\s*\["London",\s*"New York",\s*"Mumbai"\]/);
+
```

Comment:
```suggestion
assert.isUndefined(cities);
```

Reply:
having assert.isUndefined(cities) shows reference error. May I know your thoughts about assert.notMatch(code, /cities/) impling cities should not be in anywhere in code ?

Reply:
That could work, yes. Much more forgiving than the entire regex.